packages/theming/apple-theme/src/createAppleTheme.macos.ts

```diff
@@ -1,6 +1,42 @@
 import { ThemeReference } from '@fluentui-react-native/theme';
-import { defaultAppleThemeMacOS } from './appleTheme.macos';
+import { NativeEventEmitter, NativeModules } from 'react-native';
+import { BaseAppleThemeMacOS } from './appleTheme.macos';
+
+// TODO: There are some unresolved bugs with the native module, so let's use the fallback JS theme instead
+const useNativeThemingModule = false;
 
 export function createAppleTheme(): ThemeReference {
-  return new ThemeReference(defaultAppleThemeMacOS);
+  const appleThemeReference: ThemeReference = new ThemeReference(BaseAppleThemeMacOS);
+
+  if (useNativeThemingModule) {
+    const module = NativeModules && NativeModules.MSFAppleThemeModule;
+    if (module) {
+      module.getApplePartialThemeWithCallback((error, applePartialTheme) => {
+        if (error) {
+          console.error(`Error retrieving apple theming module! ${error}`);
+        }
+        // Layer the native apple theming module values on top of the base Apple theme
+
+        // TODO this doesn't work. Why?
+        // appleThemeReference.update(applePartialTheme);
+        // Set the individual properties instead
+        appleThemeReference.theme.colors = applePartialTheme.colors;
+        appleThemeReference.theme.typography = applePartialTheme.typography;
+        appleThemeReference.invalidate();
```

Comment:
Unless I'm obviously doing something wrong here, This feels like a bug. I'll update this comment with a bug number unless I or someone else finds out what's wrong beforehand